x-pack/legacy/plugins/index_lifecycle_management/__jest__/components/edit_policy.test.js

```diff
@@ -464,14 +455,14 @@ describe('edit policy', () => {
     });
   });
   describe('delete phase', () => {
-    test('should show positive number required error when trying to save delete phase with 0 for after', () => {
```

Comment:
Same comment as above. I believe this is still a valid test case, but should be updated to test with a negative number instead of `0`.

Reply:
Same response as above. There shouldn't be an error as `0` is now valid. I edited the test to check for no errors.

Reply:
Update: You can ignore my original comment. I see there is a test to check for negative numbers below this test so I think we're all set:
```
    test('should show positive number required error when trying to save delete phase with -1 for after', () => {
```